packages/ui/src/components/EventsCard/EventsCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { Box, BoxProps } from '../Box';
+import { Button } from '../Button';
+import { Link } from '../Link';
+import { Typography } from '../Typography';
+import { BUTTON_TEXT, MAIN_TITLE } from './EventsCard.mocks';
+
+export type Event = {
+  id: number;
+  title: string;
+  date: Date | { from: Date; to: Date };
+  description: string;
+  url: string;
+};
+
+export type EventCardProps = { events: Event[] } & BoxProps;
```

Comment:
💭 As a convention we should use interfaces for component props

Reply:
it's done